src/appleseed/renderer/modeling/environmentedf/oslenvironmentedf.cpp

```diff
@@ -182,11 +377,104 @@ namespace
             else value.set(0.0f);
         }
 
-        ShaderGroup* m_shader_group;
+        float compute_pdf(
+            const float   u,
+            const float   v,
+            const float   theta) const
+        {
+            assert(u >= 0.0f && u < 1.0f);
+            assert(v >= 0.0f && v < 1.0f);
+            assert(m_importance_sampler);
+
+            // Compute the probability density of this sample in the importance map.
+            std::size_t m_importance_map_width = m_importance_map_size;
```

Comment:
Don't use `m_` prefix for local variables.

Make const.

Reply:
Done for all.